### DISCUSSION TITLE
is there an area parameter?

### DISCUSSION THREAD

**Discussion by jorgesisco** [jsvine/pdfplumber#660] [Ask for help with specific PDFs]
Using the following method, I am able to extract the entire table on a single df:

      def getTablesGRI_COH4B(self):
		
		pdf = pdfplumber.open(self.file_path, pages=self.page_range)
		frames = []

		for i in range(0, len(self.page_range)):
			try:
				page = pdf.pages[i]
				table = page.extract_table()
				frames.append(pd.DataFrame(table))
			
			except:
				pass

		df =  pd.concat(frames)
		df = df.drop_duplicates()

		return df

But I am since the tables are not the exact same size, I need to specify the cells I need but I have not noticed in the docs if that's possible.

Ps. I can't share the pdf due to an NDA, I just want to know if is posible to extract the values based on designated area.

Thanks in advance 🙏

**Comment by jsvine** [comment]
Hi @jorgesisco, and thanks for your interest in this library. I'm not sure I understand the question 100%. When you say "based on designated area," do you mean the coordinate location on the page itself? If so, you might want to try `Page.crop((x0, top, x1, bottom))` . Or do you mean the particular cells in the table? If so, there's no particular `pdfplumber` feature for that; it seems like something that is probably more easily handled in your own Python code, depending on the specifics of your tables.

**Reply by jorgesisco** [reply]
Hello! Thanks for your reply.

I meant the coordinate location to be read on each page.
